train.py

```diff
@@ -93,6 +93,7 @@ def train_model(args, device, parallel):
     # make data loader
     ds_cfgs = [a for a in sorted(args.keys()) if 'dataset' in a]
     assert ds_cfgs[0] == 'dataset'
+    mean, std = args[ds_cfgs[0]]['mean'], args[ds_cfgs[0]]['std'] # read default mean and std first
```

Comment:
Added these to bound mean and std. These two are referenced in line 140 but only appear in a loop as local variables.